packages/core/strapi/lib/services/entity-service/__tests__/entity-service.test.js

```diff
@@ -366,7 +366,16 @@ describe('Entity service', () => {
 
         const fakeStrapi = {
           getModel: jest.fn((modelName) => fakeModels[modelName]),
+          query: jest.fn(() => fakeQuery),
+          db: {
+            dialect: {
+              client: 'sqlite',
+            },
+          },
         };
+
+        global.strapi = fakeStrapi;
```

Comment:
I think you can use spies on global 🤔 
```
jest
  .spyOn(global, 'strapi', 'get')
  .mockImplementation(() => strapi)
```

WDYT ?

Reply:
what is the difference in this scenario? I feel it's a little bit more verbose to use spies here

Reply:
It is to be able to restore the global state after tests are passed. In order to not corrupt the global for other tests.

Reply:
Yes, as it is rewritten at the start of the suite, and we use this in many other places I don't think we have any issue with it for the moment.
Maybe we could tackle it for all of the tests in a separate PR if we think it's an issue, wdyt?